react-a11y/index.js

```diff
@@ -1,37 +0,0 @@
-module.exports = {
-	'extends': '../react/index.js',
```

Comment:
Would it be preferable and feasible to add an extends rule for [react-a11y](https://github.com/evcohen/eslint-plugin-jsx-a11y#usage)?

```js
{
  'extends': [
    '../react/index.js',
    'plugin:jsx-a11y/recommended'
  ]
}
```

That would seem to strike a good balance as consumers wouldn't need to add the extends themselves and we could stop maintaining a ruleset ourselves.

Reply:
See https://github.com/Automattic/wp-calypso/pull/21687#discussion_r162689099 for some musings on that. I don't have a very strong opinion on this, it just seemed to make sense to separate concerns and leave it up to consumers to import the a11y ruleset themselves (much like Calypso also [imports the recommended jest ruleset directly](https://github.com/Automattic/wp-calypso/blob/6812e4b9ed2cbf65cf7ecaba2ff5fcb75e37865b/.eslintrc.js#L5)).